src/shared/test_util/tbots_gtest_main.h

```diff
@@ -15,4 +15,10 @@ struct TbotsGtestMain
 
     // Directory to send g3log logger logs
     static std::string logging_dir;
+
+
+    // Controls the speed of the simulated test. Values from 0.1 to 0.9 are the slow down
+    // factor i.e 0.1 test_speed will play test 10X slower. Values from 1.1 to 10 will
+    // play test faster. Default value is 1.
+    static double test_speed;
```

Comment:
do we have to use increments of 0.1?

Reply:
No, just any double between the set max and min

Reply:
Nit but I think we should update the comment to something like values in range (0, 1) for slowing down and (1, 10] for speeding up. This makes it clearer that it's a continuous rather than discrete range.

Reply:
Good point, I'll make the comment more discriptive

Reply:
ahh sorry it should be [0.1, 1) for slow down since the min value is 0.1 right? I missed that initially my bad

Reply:
Fixed